pkgs/development/haskell-modules/configuration-ghc-9.8.x.nix

```diff
@@ -48,27 +49,119 @@ self: super: {
   unix = null;
   xhtml = null;
 
-  # https://github.com/tibbe/unordered-containers/issues/214
-  unordered-containers = dontCheck super.unordered-containers;
+  aeson = super.aeson_2_2_1_0;
+  aeson-pretty = super.aeson-pretty_0_8_10;
+  attoparsec-aeson = super.attoparsec-aeson_2_2_0_1;
+  alex = super.alex_3_4_0_1;
+  bifunctors = super.bifunctors_5_6_1;
+  cabal-install-solver = super.cabal-install-solver_3_10_2_1;
+  doctest = super.doctest_0_22_2;
+  free = super.free_5_2;
+  # ghc-lib 9.8.1.20231121 required for Cabal to build: https://github.com/digital-asset/ghc-lib/issues/495
```

Comment:
```suggestion
  # ghc-lib 9.8.1.20231121 required so that correct Cabal files are generated: https://github.com/digital-asset/ghc-lib/issues/495
```